app/src/main/java/com/allan/boardbuddies/EditActivity.java

```diff
@@ -3,23 +3,75 @@
 import androidx.appcompat.app.AppCompatActivity;
 import androidx.appcompat.widget.Toolbar;
 
+import android.content.Context;
+import android.content.Intent;
 import android.os.Bundle;
 import android.view.View;
+import android.view.inputmethod.InputMethodManager;
 import android.widget.EditText;
-import android.widget.ScrollView;
+import org.json.JSONException;
+import org.json.JSONObject;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
 
 public class EditActivity extends AppCompatActivity {
+    private EditText editTextTitle;
+    private EditText editTextContent;
+    private Bundle extras;
```

Comment:
recommend storing the title and content as private string fields instead of storing the whole thing as "extras", which doesn't really tell the reader what's in it